lib/features/tabs/ui/screens/components/top_picks_item.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:elvan/shared/components/constants/app_colors.dart';
+import 'package:elvan/shared/components/constants/strings.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/container.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class TopPicksItem extends StatelessWidget {
+  const TopPicksItem({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
```

Comment:
Make a Generic Component in the shared/components/card folder. There should be three different cards. One for categories, one for top picks, and one for recent orders.